express-graphql/dist/renderGraphiQL.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.renderGraphiQL = renderGraphiQL;
+
+
+// Current latest version of GraphiQL.
+var GRAPHIQL_VERSION = '0.10.1';
+
+// Ensures string values are safe to be used within a <script> tag.
+
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+function safeSerialize(data) {
+  return data ? JSON.stringify(data).replace(/\//g, '\\/') : 'undefined';
+}
+
+
+const introductionDemoQuery = `# Welcome to 1BahnQL
+#
+# GraphiQL is an in-browser IDE for writing, validating, and
+# testing GraphQL queries.
+#
+# Type queries into this side of the screen, and you will
+# see intelligent typeaheads aware of the current GraphQL type schema and
+# live syntax and validation errors highlighted within the text.
+#
+# To bring up the auto-complete at any point, just press Ctrl-Space.
+#
+# Press the run button above, or Cmd-Enter to execute the query, and the result
+# will appear in the pane to the right.
+#
+# Learning resources:
+# GraphQL: http://graphql.org
+# GraphiQL: https://github.com/graphql/graphiql
+# 1BahnQL: https://github.com/dbsystel/1BahnQL
+#
+#
+# Example queries:
+# Just comment out the query you would like to test and press the run button above,
+# or Cmd-Enter to execute the query
+
+{
+  stationWithEvaId(evaId: 8000105) {
+    name
+    location {
+      latitude
+      longitude
+    }
+    picture {
+      url
+    }
+  }
+}
+
+# More complex query examples:
+#
+# Querys 5 stations in a distance of 2000m with their corresponding facilities and location
```

Comment:
Queries 5 stations within a distance of 2000m and their corresponding facilities and location